auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -286,6 +286,8 @@ export default defineComponent({
             statusText = '<strong>Not Authorized</strong>. Your request to manage this business has been declined.'
             break
           case AffiliationInvitationStatus.Expired:
+            statusText = 'Invitation is <strong>Expired</strong>.'
```

Comment:
@hfekete added this in, will have to check with Ethan if it's correct - I don't think yours expires?